pkg/formatted/color.go

```diff
@@ -48,6 +48,8 @@ func DecorateAttr(attrString, message string) string {
 		return fmt.Sprintf("📦 ")
 	case "params":
 		return fmt.Sprintf("⚓ ")
+	case "timeout":
```

Comment:
It's a shame because it's a perfect emoji, but we can probably leave the emoji out in this case and follow the pattern for ServiceAccount in the Template.

Reply:
Agreed, thanks, too many emojis breaks the charm of the emojis !

Reply:
Makes sense, removed the emoji.